js-miniapp-sample/src/pages/event-listener.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { MiniAppEvents } from 'js-miniapp-sdk';
+
+import {
+  Button,
+  CardContent,
+  CardActions,
+  makeStyles,
+} from '@material-ui/core';
+
+import GreyCard from '../components/GreyCard';
+
+const useStyles = makeStyles((theme) => ({
+  scrollable: {
+    overflowY: 'auto',
+    width: '100%',
+    paddingTop: 20,
+    paddingBottom: 20,
+  },
+  card: {
+    width: '100%',
+    height: 'auto',
+  },
+  actions: {
+    justifyContent: 'center',
+  },
+  content: {
+    justifyContent: 'center',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    fontSize: 18,
+    color: theme.color.primary,
+    fontWeight: 'bold',
+    paddingBottom: 0,
+  },
+  info: {
+    fontSize: 16,
+    lineBreak: 'anywhere',
+    wordBreak: 'break-all',
+    color: theme.color.primary,
+    marginTop: 0,
+    paddingBottom: 10,
+  },
+}));
+
+let externalwebviewcloseeventcount = 0;
+let pauseeventcount = 0;
+let resumeeventcount = 0;
+
+window.addEventListener(MiniAppEvents.externalwebviewclose, function (e) {
+  let message = e.detail;
+  console.log(message);
+  externalwebviewcloseeventcount++;
+  const webviewclose = document.getElementById('webviewclose');
+  if (webviewclose) {
+    webviewclose.innerText =
+      'External Webview Closed: ' + externalwebviewcloseeventcount;
+  }
+});
+
+window.addEventListener(MiniAppEvents.pause, function (e) {
+  let message = e.detail;
+  console.log(message);
+  pauseeventcount++;
+  const pause = document.getElementById('pause');
+  if (pause) {
+    pause.innerText = 'Mini App Paused: ' + pauseeventcount;
```

Comment:
These values should be updated reactively. I think that's why you're getting errors on CI: you aren't really supposed to directly manipulate the DOM when using React.

You can use state hooks for this, see some of the other screens for an example, i.e. https://github.com/rakutentech/js-miniapp/blob/master/js-miniapp-sample/src/pages/local-storage.js#L48

So in your case, you would do something like:
```
const [pauseEventCount, setPauseEventCount] = useState(0);

window.addEventListener(MiniAppEvents.pause, function (e) {
  setPauseEventCount(pauseeventcount++);
}

// And use directly in the JSX
<p>Mini App Paused: {pauseEventCount}</p>
```